### ISSUE TITLE
Make saving async

### ISSUE THREAD

**Issue by lgessler** [gucorpling/gitdox#84]
Saving causes a page refresh, which loses the user's place in a document. Saves should be made asynchronous with an XHR, and a toast notification should let the user know whether the save was successful or not.

**Comment by amir-zeldes** [comment]
True, but low priority, since most work time is spent in spreadsheet mode, which is live
